RealtimeStreamToImpala/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "template":"etlBatch",
+  "config":{
+    "schedule":"*/5 * * * *",
```

Comment:
in the description you said every 10 minutes, while this looks like 5 mins, am i missing anything ?